markdown/pxf/JdbcPXF.html.md.erb

```diff
@@ -0,0 +1,213 @@
+
+| Keyword  | Value |
+|-------|-------------------------------------|
+| \<column-name\>    | The name of the PXF external table column. The PXF \<column-name\> must exactly match the \<column-name\> used in the external SQL table.|
+| \<data-type\>    | The data type of the PXF external table column. The PXF \<data-type\> must be equivalent to the data type used for \<column-name\> in the SQL table.|
+| \<host\>    | The PXF host. While \<host\> may identify any PXF agent node, use the HDFS NameNode as it is guaranteed to be available in a running HDFS cluster. If HDFS High Availability is enabled, \<host\> must identify the HDFS NameService. |
+| \<port\>    | The PXF port. If \<port\> is omitted, PXF assumes \<host\> identifies a High Availability HDFS Nameservice and connects to the port number designated by the `pxf_service_port` server configuration parameter value. Default is 51200. |
+| \<jdbc-schema-name\>    | The schema name. The default schema name is `default`. |
+| \<jdbc-database-name\>    | The database name. The default database name is determined by the external SQL server. |
+| \<jdbc-table-name\>    | The table name. |
+| PROFILE    | The `PROFILE` keyword must specify `Jdbc`. |
+| \<custom-option\>  | The custom options supported by the `Jdbc` profile are discussed later in this section.|
+| FORMAT 'CUSTOM' | The JDBC `CUSTOM` `FORMAT` supports only the built-in `'pxfwritable_import'` `FORMATTER` property. |
+
+*Note*: When creating PXF external tables, you cannot use the `HEADER` option in your `FORMAT` specification.
+
+
+### <a id="profile_jdbcoptipns"></a>JDBC Custom Options
+
+You may include one or more custom options in the `LOCATION` URI. Preface each option with an ampersand `&`. 
+
+The JDBC plug-in `Jdbc` profile supports the following \<custom-option\>s:
+
+| Option Name   | Description
+|---------------|--------------------|
+| JDBC_DRIVER | The JDBC driver class name. |
+| DB_URL | The URL to the database; includes the hostname, port, and database name. |
+| USER | The database user name. |
+| PASS | The database password for USER. |
+| PARTITION_BY | The partition column, \<column-name\>:\<column-type\>. The JDBC plug-in supports `date`, `int`, and `enum` \<column-type\>s. Use the  `yyyy-MM-dd` format for the `date` \<column-type\>. A null `PARTITION_BY` defaults to a single fragment. |
+| RANGE | The query range, \<start-value\>[:\<end-value\>]. \<end-value\> may be empty for an `int` \<column-type\>. |
+| INTERVAL | The interval, \<interval-num\>[:\<interval-unit\>], of one fragment.  `INTERVAL` may be empty for an `enum` \<column-type\>. \<interval-unit\> may be empty for an `int` \<column-type\>. |
+
+**Note**: The partition `RANGE` interval is left closed, right open. That is, the range includes the \<start-value\> but does *not* include the \<end-value\>.
```

Comment:
I'd move this (not as a note, necessarily, as MD formatting won't work) to the RANGE description in the table.